openedxstats/apps/sites/tests.py

```diff
@@ -375,3 +379,33 @@ def test_site_get_geographies_method_with_unicode(self):
         self.assertEqual(str(sitegeozone1), "https://www.κόσμε.co---Greece")
         self.assertEqual(str(geozone2), "\u00e9")
 
+
+class OTChartTestCase(TestCase):
+    def setUp(self):
+        user = User.objects.create_user('testuser', 'testuser@edx.com', 'password')
+        if user is None:
```

Comment:
Yeah that was a check for when I was using authenticate(), but I accidentally left in the check. I removed it now